.github/update-release-branch.py

```diff
@@ -122,7 +124,7 @@ def get_commit_difference(repo):
 
 # Is the given commit the automatic merge commit from when merging a PR
 def is_pr_merge_commit(commit):
-  return commit.committer.login == 'web-flow' and len(commit.parents) > 1
+  return commit.committer != None and commit.committer.login == 'web-flow' and len(commit.parents) > 1
```

Comment:
I prefer this over equality checks.
```suggestion
  return commit.committer is not None and commit.committer.login == 'web-flow' and len(commit.parents) > 1
```